### PR TITLE
1.4/1653 update links for policies in the app

### PR DESCRIPTION
**Pull request by AlinChindea**
Trello 🎫 [https://trello.com/c/GdjPZpbX/1653-update-links-for-policies-in-the-app-vvv-ee](https://trello.com/c/GdjPZpbX/1653-update-links-for-policies-in-the-app-vvv-ee)

### CODE REVIEW

File: tests/pages/login.test.js [hunk 1]
```diff
@@ -34,7 +49,7 @@ test("Can log in", async (t) => {
     expect(pagePath).to.equal(url + "/");
 });
 
-test("Can log out", async (t) => {
+test.only("Can log out", async (t) => {
```
Comment (AlinChindea) [inline tests/pages/login.test.js]:
```suggestion
test("Can log out", async (t) => {
```

File: src/App.vue [hunk 2]
```diff
@@ -2,16 +2,25 @@
...
 export default {
     name: "app",
     components: {
         Masthead,
+        PrivacyLink
     },
+    computed: {
+        footerShow: function () {
+            const paths = ["/", "/scan"];
+            return !(paths.indexOf(this.$route.path) === -1);
```
Comment (charlesstrange2) [inline src/App.vue:21]: nb.
if we ever to es2016, it has `[].inlcudes()`

File: tests/pages/login.test.js [hunk 3]
```diff
@@ -46,6 +60,8 @@ test("Can log out", async (t) => {
     await t.click(logout);
     const pagePath = await t.eval(() => document.documentURI);
     expect(pagePath).to.equal(url + "/login");
+    const privacyLink = await el("#privacy").exists;
```
Comment (charlesstrange2) [inline tests/pages/login.test.js:63]: a little cheeky, hijacking the logout page to see the privacy policy; 
might have been better to have an "i can see the privacy policy link on the logout page" test

**Review by charlesstrange2 — APPROVED** [review]
I think I'd have preferred the logout page privacy link to be a separate test but it works

File: src/App.vue (hunk 2 above)
Comment (AlinChindea) [reply src/App.vue:21]: not supported by IE - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Browser_compatibility

File: tests/pages/login.test.js (hunk 3 above)
Comment (AlinChindea) [reply tests/pages/login.test.js:63]: wouldn't that just duplicate teh whole `Can log out`test and just append two more lines of code? the ones I've added here. Also, when you log out you're redirected to the login page which already contains separate tests about the privacy link. come to think about it, it's redundant to add it even in the "Can log out" test as long as we redirect to login on logout.

**Pull request merged**